### ISSUE TITLE
Recreate Dashboard - Drivers

### ISSUE THREAD

**Issue by peterfrenchons** [GSS-Cogs/ukstats.ccv2.theme#12]
- [x] Chart 1: Primary energy demand, UK, 1995 to 2020, million tonnes of oil equivalent (Mtoe)
- [x] Chart 2: Fossil fuel energy use for the four highest sectors in the UK, 1990 to 2019, (million tonnes of oil equivalent)
- [x] Chart 3: Energy intensity indices relative to the year 2000, UK, 2000 to 2020

**Comment by Shannon95** [comment]
3 Driver charts re-created and ready for review. 
https://staging.climate-change.data.gov.uk/dashboards/drivers

**Comment by peterfrenchons** [comment]
Observations Chart 1:
Y axis data points for some Energy types differ v1.1 vs v2.0. Just an observation
For Nuclear the upper y axis value 24 not displayed.

Observations: Chart 2
Y axis on v1.1 starts at 0, v2.0 higher
Y axis on v1.1 does not change on choice of indicator
Data includes 2020 on v2.0, 2019 on v1.1. Need to ensure CC team sign off for 2020

Observations: Chart 3
Y axis data points for Road are 2, others = 5. Just an observation

General observations:
Text in figure before the chart should be in same font size as footnotes?
Hover values v2.0 Year, Label, value. v1.1 (Year, Value),Label. Trailing zeros not removed. Just an observation

**Comment by Shannon95** [comment]
Thanks for the review/feedback @peterfrenchons. I've updated a few aspects to reflect the comments. Let me know if you have any more feedback. Thanks. 
Chart 1:
- Added 50px Top margin. This has allowed the upper Y axis to be shown as in  v1.1. e.g. For Nuclear, the upper y-axis value of 24 is now displayed.
- Trailing zeros removed where applicable. 

Chart 2: 
- Changed Range mode to tozero, this allows the y-axis to begin at 0. 
-  y-axis does change for me on v1.1 and v2.0 on the choice of indicator
- An update has been done to include the 2020 data on v1.1 which a PR I committed has been merged in. It is visible [here](https://beta.climate-change.data.gov.uk/drivers#chart-container-drivers-02) but needs approval for a new release for it to show on the actual V1.1 site. [Here](https://github.com/GSS-Cogs/climate-change-govuk/issues/430) is the issue related. 
- Trailing zeros removed where applicable. 

Chart 3
- y-axis data points for Road are is separated in counts of 5 the same as the other options. I Added 50px to the bottom margin which fixed this. 
- Trailing zeros removed where applicable.
